docs/reference/data-frames/apis/get-transform-stats.asciidoc

```diff
@@ -54,6 +60,16 @@ see {stack-ov}/security-privileges.html[Security privileges] and
 
 ==== Examples
 
+The following example skips for the first five {dataframe-transforms} and
+gets usage information for a maximum of ten results: 
+
+[source,js]
+--------------------------------------------------
+GET _data_frame/transforms/_stats?from=5;size=10
```

Comment:
Should the `;` between from and size be a `&` here?

Reply:
It worked in Kibana, but I'll test again